src/main/java/redis/clients/jedis/BinaryClient.java

```diff
@@ -100,9 +102,10 @@ public void set(final byte[] key, final byte[] value) {
     sendCommand(Command.SET, key, value);
   }
 
-  public void set(final byte[] key, final byte[] value, final byte[] nxxx, final byte[] expx,
-      final long time) {
-    sendCommand(Command.SET, key, value, nxxx, expx, toByteArray(time));
+  public void set(final byte[] key, final byte[] value, final byte[] nxxx, final byte[] expx, final byte[] time) {
```

Comment:
I agree with that.

This way we will be more flexible if `*Command*Parameters` have any changes. I'll fix it.